src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1965,7 +1964,9 @@ protected override void BeginProcessing()
 
                 startInfo.WindowStyle = _windowstyle;
 
-                if (_nonewwindow)
+                // When starting a process as another user, the 'CreateNoWindow' property value is ignored and a new window is created.
+                // See details at https://learn.microsoft.com/dotnet/api/system.diagnostics.processstartinfo.createnowindow?view=net-9.0#remarks
+                if (_nonewwindow && _credential is null)
                 {
                     startInfo.CreateNoWindow = _nonewwindow;
                 }
```

Comment:
Can we use more readable code:
```c#
startInfo.CreateNoWindow = _nonewwindow && _credential is null;
```

Reply:
Given that we are backporting this PR, I'm a little hesitated to always set this property. I'd rather to keep it as is.